node/http.go

```diff
@@ -96,7 +102,7 @@ func makeRPCOverHTTP(host string, port uint16, servers map[string]*jsonrpc.Serve
 }
 
 func makeRPCOverWebsocket(host string, port uint16, servers map[string]*jsonrpc.Server,
-	log utils.SimpleLogger, metricsEnabled bool, corsEnabled bool,
+	log utils.SimpleLogger, metricsEnabled bool, corsEnabled bool, rpchandler *rpc.Handler,
```

Comment:
the reason why we need `/ready` and `/ready/sync` paths are readiness checks in kubernetes kind of environment. There is no need for now to have it on websocket. So I propose to revert this changes for websockets

Reply:
Removed it in 294c5eefcfc3ecd6465dbde1be62b0d5f8497314